examples/microwave-oven-app/microwave-oven-common/include/microwave-oven-device.h

```diff
@@ -201,6 +240,15 @@ class ExampleMicrowaveOvenDevice : public MicrowaveOvenControl::Delegate,
     ModeBase::Instance mMicrowaveOvenModeInstance;
     MicrowaveOvenControl::Instance mMicrowaveOvenControlInstance;
 
+    // MicrowaveOvenControl variables
+    BitMask<MicrowaveOvenControl::Feature> kFeature = to_underlying(MicrowaveOvenControl::Feature::kPowerAsNumber) +
```

Comment:
Casting feels not great in type-safe code. since this is named `k....` this should be const.


Bitmask has a multi-value constructor and this would work:
 
 ```cpp
const BitMask<MicrowaveOvenControl::Feature> kFeature(
    MicrowaveOvenControl::Feature::kPowerAsNumber,
    MicrowaveOvenControl::Feature::kPowerNumberLimits)
```

however I am not sure if that works in a header file. Does this need to be in the header? Could we just have it as a const inside the cpp file?